lib/awful/mouse/init.lua

```diff
@@ -35,28 +35,29 @@ mouse.wibox = {}
 --
 -- @DOC_screen_client_snap_EXAMPLE@
 --
--- @tfield integer awful.mouse.snap.default_distance
+-- @tfield integer snap.default_distance
```

Comment:
LDoc adds a `mouse.` prefix to that name, so the current docs show `mouse.awful.mouse.snap.client_enabled`.
I don't know how to make it only drop that prefix, so I settled on the current result `mouse.snap.client_enabled` instead.

Reply:
We had the same problem with signals and methods a while ago. For one, I made some magic in `config.ld` and managed to fix it. For the other, I made a new tag type rather than use the built-in ones. This way you get more control.

Reply:
So, we need to hack the `config.ld` to make ldoc recognize correctly module path with the `@tfield` tag?

Then, let's keep the current version and fix that with another PR.